connectivity/nfc/CMakeLists.txt

```diff
@@ -1,9 +1,15 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+include(${MBED_ROOT}/tools/cmake/components.cmake)
+
+add_library(mbed-os-nfc STATIC)
```

Comment:
I've tested - this could be static or object. as we use weak symbols we could start now with objects so no breakages within Mbed OS

App can link with external componetns with static, should work.

Reply:
Lets use object for these internal components